test/functionalTest/README.md

```diff
@@ -1,6 +1,55 @@
 ## How to run functional tests
 
-TBD
+### Prerequisites
+
+* The `contextBroker` binary is in execution path
+* MongoDB database is up and running
+* accumulator-server.py is correctly installed and available in the execution path (see specific section below)
+
+### How to install accumulator-server.py
+
+We recommend to use Python [virtualenv](https://virtualenv.pypa.io/en/latest) to install the required dependencies to avoid any potential conflict with operating system Python installation. So, first create your virtual env (named for instance `ft_env`):
+
+```
+pip install virtualenv  # if you doesn't have virtualenv itself previously installed
```

Comment:
doesn't -> don't
itself -> {} (just remove the word)

Reply:
Ok, the first, no the second as I want to emphatise on virtualenv. I mean, "si necesitas instalar el propio virtualenv previamente" vs "si necesitas instalar virtualenv previamente".

Reply:
Fixed in 68b58b8